starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx

```diff
@@ -5,7 +5,7 @@ import PageFooter from '~/components/PageFooter';
 export default function ApiExample() {
   return (
     <>
-      <PageHeader>Solid Start Tanstack Query Fetch Data from API</PageHeader>
+      <PageHeader>Solid Start Fetch Data from API</PageHeader>
```

Comment:
I think ken meant `SolidStart` without the space

Reply:
I'm sooooo stupid 😅

Reply:
Not really.
I actually thought the same thing you did